runtime/compiler/CMakeLists.txt

```diff
@@ -48,6 +48,18 @@ if(OMR_ARCH_X86)
 		"-I${CMAKE_CURRENT_SOURCE_DIR}/x/i386/runtime/"
 	)
 	omr_append_flags(CMAKE_ASM_NASM_FLAGS ${asm_inc_dirs})
+	# For whatever reason cmake does not apply compile definitions when building nasm objects
+	# The if guard is here in case they ever start doing so
+	if(NOT CMAKE_ASM_NASM_COMPILE_OBJECT MATCHES "<DEFINES>")
+		# Tack on defines immediately following the compiler name
```

Comment:
Please complete sentences with a period.

Reply:
The comment on line 54 is not fixed.

Reply:
@dnakamura pls add the other period so we can get this merged.